mapie/tests/test_control_risk.py

```diff
@@ -0,0 +1,161 @@
+"""
+Testing for control_risk module.
+Testing for now risks for multilabel classification
+"""
+import numpy as np
+# from numpy.typing import NDArray
+from mapie.control_risk.risks import (_compute_precision,
+                                      _compute_recall)
+import pytest
+from typing import Union, List
+from numpy.typing import NDArray
+from mapie.control_risk.p_values import hoefdding_bentkus_p_value
+from mapie.control_risk.ltt import (_ltt_procedure,
+                                    _find_lambda_control_star)
+
+
+lambdas = np.array([0.5, 0.9])
+y_toy = np.stack([
+    [1, 0, 1],
+    [0, 1, 0],
+    [1, 1, 0],
+    [1, 1, 1],
+])
+y_preds_proba = np.stack([
+    [0.2, 0.6, 0.9],
+    [0.8, 0.2, 0.6],
+    [0.4, 0.8, 0.1],
+    [0.6, 0.8, 0.7]
+])
+y_preds_proba = np.expand_dims(y_preds_proba, axis=2)
+test_recall = np.array([
+    [1/2, 1.],
+    [1., 1.],
+    [1/2, 1.],
+    [0., 1.]
+])
+test_precision = np.array([
+    [1/2, 1.],
+    [1., 1.],
+    [0., 1.],
+    [0., 1.]
+])
+r_hat = np.array([
+    0.5, 0.8
+])
+n = 1100
+alpha = np.array([0.6])
+valid_index = [[
+    0, 1
+]]
```

Comment:
STYLE: add newline to separate these different elements in your code for visibility.

Reply:
Done